boosty/types/media_types.py

```diff
@@ -103,7 +103,6 @@ class Audio(FileBase):
 
 class Video(FileBase):
     type: Literal["ok_video"]
-    url: HttpUrl | Literal[""]
```

Comment:
Why did you remove this line? Also, could you let me know why you removed only this line, but left the comment beneath it?

Reply:
Because I changed the type of  `FileBase.url` field to `HttpUrl | Literal[""]`.
I accidentally left the comment, my bad.